README.md

```diff
@@ -1,2 +1,58 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
 # Hello-Microverse-Project
-In this project, I set up a "Hello world" repository. 
+
+> In this project, I set up a "Hello world" repository.
+
+## Built With
+
+- HTML and CSS
+- Without Frameworks
+
+## Getting Started
+
+**This is first little project on Microverse.**
+**The Learning Objectives are:**
+
+- Create branches with different base branches.
+- Understand the concept of version control.
+- Follow GitHub flow.
+- Open pull requests in GitHub.
+- Write descriptive, easy to understand commit messages.
+- Write short, detailed, easy to understand descriptions of pull requests.
+- Write a descriptive and easy to understand README in business English for every project submitted.
+- Demonstrate an ability to submit a project for code review.
+
+### Prerequisites
+
+### Setup
+
+### Install
+
+### Usage
+
+### Run tests
+
+### Deployment
```

Comment:
- [OPTIONAL] Make sure to remove all unnecessary titles since they are empty (Prerequisites, run tests Deployment ....)